partcad/src/partcad/assembly_factory_assy.py

```
@@ -64,33 +65,24 @@
                 fp.close()
 
                 # Resolve Jinja templates
-                template = Environment(
-                    loader=FileSystemLoader(
-                        os.path.dirname(self.path) + os.path.sep
-                    )
-                ).from_string(config)
+                template = Environment(loader=FileSystemLoader(os.path.dirname(self.path) + os.path.sep)).from_string(
```

Ruff (0.8.2)

By default, jinja2 sets `autoescape` to `False`. Consider using `autoescape=True` or the `select_autoescape` function to mitigate XSS vulnerabilities.

(S701)